src/Core/Managed/Shared/Extensibility/Implementation/TaskTimer.cs

```diff
@@ -77,7 +81,33 @@ public void Start(Func<Task> elapsed)
                             // It should return Task.FromResult but just in case we check for null if someone returned null
                             if (task != null)
                             {
+#if !NET40
                                 await task.ConfigureAwait(false);
+#else
+                                task.ContinueWith(
+                                    task1 =>
+                                    {
+                                        try
+                                        {
+                                            task1.Wait();
```

Comment:
you use Wait to hause exceptions to propagate. Elsewhere you use RethrowIf.... for that. Why the difference?